src/server/browserType.ts

```diff
@@ -45,7 +45,7 @@ export abstract class BrowserType extends SdkObject {
   }
 
   executablePath(channel?: string): string {
-    return registry.executablePath(this._name) || '';
+    return registry.findExecutable(this._name).executablePathIfExists() || '';
```

Comment:
This is functional change.

Today, when installing `@playwright/test` without running `npm playwright install`, the `chromium.executablePath()` will return a value. With this change, it will return empty string until the browsers are installed.

Do you think it's worth keeping the current behavior just to be on the safe side?

Reply:
Good catch! My next PR fixes this, so I'll port the changes here.